anta/cli/nrfu/commands.py

```diff
@@ -63,6 +60,28 @@ def text(ctx: click.Context) -> None:
     exit_with_code(ctx)
 
 
+@click.command()
+@click.pass_context
+@click.option(
+    "--csv-output",
+    type=click.Path(
+        file_okay=True,
+        dir_okay=False,
+        exists=False,
+        writable=True,
+        path_type=pathlib.Path,
+    ),
+    show_envvar=True,
+    required=False,
+    help="Path to save report as a CSV file. It only saves test results and not the output from --group-by option",
```

Comment:
Do we still need "It only saves test results and not the output from --group-by option" with your latest changes?

Reply:
It is still the case: we don't save `--group-by` results. So the warning message